homeassistant/components/teslemetry/strings.json

```diff
@@ -0,0 +1,35 @@
+{
+  "config": {
+    "error": {
+      "invalid_access_token": "[%key:common::config_flow::error::invalid_access_token%]",
+      "subscription_required": "Subscription required",
```

Comment:
Maybe we can give some more info about the subscription?

Reply:
Or add a description to the step where we explain what kind of subscription you need

Reply:
The end state is to have a repair action, but I'll make this more instructive.